### PR TITLE
Add Birthday field to Person

### PR DESCRIPTION
**Pull request by czhongwei**
- Add Birthday field to Person
- Add testcases in BirthdayTest.java
- Include parsing of Birthday in all AddCommandParser, EditCommandParser and ParserUtil
- Amend existing testcases to include Birthday, which includes tests for storage, model, parser, logic and command

### CODE REVIEW

**Comment by czhongwei** [comment]
Also editted UG to include birthday

**Review by rui-han-crh — CHANGES_REQUESTED** [review]
The app cannot start with the current data file because it's not in the correct format, can you change the default data to fit the current state?

**Review by rui-han-crh — APPROVED** [review]

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -52,6 +54,7 @@ public PersonCard(Person person, int displayedIndex) {
         phone.setText(person.getPhone().value);
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
+        birthday.setText(person.getBirthday().value);
```
Comment (rui-han-crh) [inline src/main/java/seedu/address/ui/PersonCard.java:57]: *

File: src/main/resources/view/PersonListCard.fxml [hunk 2]
```diff
@@ -31,6 +31,7 @@
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+      <Label fx:id="birthday" styleClass="cell_small_label" text="\$birthday" />
```
Comment (rui-han-crh) [inline src/main/resources/view/PersonListCard.fxml:34]: *

**Review by rui-han-crh — CHANGES_REQUESTED** [review]
Can you discard changes from these two files, they'll cause a merge conflict with UI. I'll fit it differently

**Review by rui-han-crh — APPROVED** [review]

**Pull request merged**